app/javascript/entrypoints/pdc/work_edit_file_upload.es6

```diff
@@ -63,7 +63,8 @@ export default class WorkEditFileUpload {
       headers: { 'X-CSRF-Token': tokenContent },
       bundle: true, // upload all selected files at once
       formData: true, // required when bundle: true
-      getResponseData() {
+      getResponseData(data) {
+        console.log(data);
```

Comment:
This shows the error:

```
<html>
<head><title>Request Rejected</title></head>
<body>The requested URL was rejected. Please consult with your administrator.<br>
<br>
Your support ID is: 12465273342320109191<br>
<br>
<a href='javascript:history.back();'>[Go Back]</a>
</body>
</html>
```

Wish whomever is returning it will also return an HTTP Status Code that is not 200

Reply:
`data` is an empty string for a successful upload.